test/statement_test.exs

```diff
@@ -7,7 +7,7 @@ defmodule StatementTest do
 
```

Comment:
could you add a test that would fail a timeout?
something like `fetch_all!(1)` and catch the exception?

Reply:
I can try. I couldn't find a good way to ensure a Sqlite3 takes at least a millisecond when I looked. Do you have something in mind?

Reply:
I found a query to use. I ran the test several times and it seemed reliable.